lib/src/main/kotlin/ca/aqtech/mctreesearch4j/Solver.kt

```diff
@@ -26,6 +60,10 @@ abstract class Solver<ActionType, NodeType: Node<ActionType, NodeType>> (
         }
     }
 
+    /**
+     * Runs a single iterations of MCTS. The default implementation runs [select], [expand], [simulate], [backpropagate]
+     * in sequence.
```

Comment:
Potentially add tot the comments that this may be modified to improve MCTS performance on different domains.